tf2onnx/tfonnx.py

```diff
@@ -18,12 +18,15 @@
 
 import tf2onnx
 import tf2onnx.onnx_opset  # pylint: disable=unused-import
+import tf2onnx.tflite_handlers  # pylint: disable=unused-import
```

Comment:
can we move the import of the tflite_handlers into 
if tflite
so we pull those only in for tflite models? From the code way below it looks like that should be possible.

Reply:
Can do